app/services/katello/repository_type_manager.rb

```diff
@@ -140,6 +140,14 @@ def enabled?(repository_type)
         find(repository_type).present?
       end
 
+      def generic_remote_options
+        options = []
+        @enabled_repository_types.each do |_, type|
```

Comment:
Since my PR for auto-detecting enabled repo types merged, this should now just be `enabled_repository_types`.